src/controls/Widget.ts

```diff
@@ -1274,16 +1275,19 @@ export class Widget_choice      <T extends { [key: string]: Widget }> implements
     }
     get serial(): Widget_choice_serial<T> {
         const out: { [key: string]: any } = {}
-        for (const key in this.state.values) out[key] = this.state.values[key].serial
+        for (const key in this.state.values) {
+            if (key !== this.state.pick) continue
```

Comment:
needs teaking; good idea, need to switch to some lazy init for perf

Reply:
There is a bug that after the form is reloaded, the choice cannot be changed anymore

Reply:
I'll merge right now, but this one is pretty important to figure out before next dev branch is merged (a.k.a tomorrow 😄)